### ISSUE TITLE
GGMAX - Limb detection

### ISSUE THREAD

**Issue by AvengingEagle** [TheGameCreators/GameGuruRepo#5274]
g_Entity[e]['limbhit'] returns the name of the last limb to be hit. Except, at the moment, there are only three possible outcomes, regardless of which limb is actually hit:
- Bip01_Head
- Bip01_Pelvis
- Bip01_R_Foot

Obviously I don't think we need every single bone to be able to register a hit, but the arms and legs should definitely register as different from the body, and from one another.

https://youtu.be/U6kUQnNlUnQ

**Comment by Ghost-l0l** [comment]
Agree, I remember that in FPS Creator it was possible - and it recognized all bones - but because of very high performance hit, nobody used it (me included).
But, it worked pretty well on (almost) empty maps :)
So, If you thinking about it, then, as AvengingEagle said, please don't overkill it. In most gamedev scenarios it would be great to have different hitboxes for head, arms, torso and legs (maybe upper and lower legs).
